tests/libvmi/factory.go

```diff
@@ -34,6 +34,20 @@ const (
 // NewFedora instantiates a new Fedora based VMI configuration,
 // building its extra properties based on the specified With* options.
 func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	return newFedora(cd.ContainerDiskFedora)
+}
+
+// NewSriovFedora instantiates a new Fedora based VMI configuration,
+// building its extra properties based on the specified With* options, the
+// image used include Guest Agent and some moduled needed by SRIOV.
+func NewSriovFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	return newFedora(cd.ContainerDiskFedoraSRIOVLane)
```

Comment:
I think you need to pass `opts` here as well:
`return newFedora(cd.ContainerDiskFedoraSRIOVLane, opts)`

Reply:
Second time I end with the same issue, thanks Alon !!!